backend/spec/requests/memos_spec.rb

```diff
@@ -70,9 +70,10 @@
   end
 
   describe 'POST /memos' do
-    context 'ログイン中かつタイトルとコンテンツが有効な場合' do
+    context 'ログイン中かつタイトルとコンテンツとユーザ名が有効な場合' do
       let(:valid_memo_params) do
-        { title: Faker::Lorem.sentence(word_count: 3), content: Faker::Lorem.paragraph(sentence_count: 5) }
+        { title: Faker::Lorem.sentence(word_count: 3), content: Faker::Lorem.paragraph(sentence_count: 5),
+          user_name: Faker::Name.name }
```

Comment:
idiomatic/Nits

改行が恣意的なので、慣習的になるようにしましょう.
```suggestion
        { title: Faker::Lorem.sentence(word_count: 3),
          content: Faker::Lorem.paragraph(sentence_count: 5),
          user_name: Faker::Name.name }
```